Fabric.Authorization.Persistence.SqlServer/Stores/SqlServerPermissionStore.cs

```diff
@@ -204,15 +212,15 @@ public SqlServerPermissionStore(IAuthorizationDbContext authorizationDbContext)
             };
         }
 
-        /// <summary>
-        /// TODO: consolidate this method and same method in SqlServerUser store
-        /// </summary>
-        /// <param name="id"></param>
-        /// <returns></returns>
-        private static string[] SplitUserId(string id)
+        private static string[] SplitGranularPermissionId(string id)
         {
-            var delimiter = new[] { @"\" };
-            return id.Split(delimiter, StringSplitOptions.RemoveEmptyEntries);
+            var delimiter = new[] { @":" };
+            var idParts = id.Split(delimiter, StringSplitOptions.RemoveEmptyEntries);
+            if (idParts.Length != 2)
+            {
```

Comment:
Shouldn't the message be `{identityProvider}:{subjectId}`?

Reply:
No - for the granular permission the id is subjectid:identityprovider - that was one of the bugs i had to fix.

Reply:
Actually, good catch - I had it incorrect, it should have been {subjectId}:{identityProvider}, I've made the change.